stdlib/public/runtime/ThreadLocalStorage.h

```diff
@@ -53,9 +53,14 @@ extern "C" int pthread_key_init_np(int key, void (*destructor)(void *));
 # ifndef __PTK_FRAMEWORK_SWIFT_KEY1
 #  define __PTK_FRAMEWORK_SWIFT_KEY1 101
 # endif
+# ifndef __PTK_FRAMEWORK_SWIFT_KEY2
+#  define __PTK_FRAMEWORK_SWIFT_KEY2 102
+# endif
+
 
 # define SWIFT_RUNTIME_TLS_KEY __PTK_FRAMEWORK_SWIFT_KEY0
 # define SWIFT_STDLIB_TLS_KEY __PTK_FRAMEWORK_SWIFT_KEY1
+# define SWIFT_RUNTIME2_TLS_KEY __PTK_FRAMEWORK_SWIFT_KEY2
```

Comment:
Please give this a more descriptive name, like `SWIFT_COMPATIBILITY_50_TLS_KEY`. That'll indicate when it's okay to reuse it.